e2e/package.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Is this file really needed? I think "node <file>" works without package.json.

Reply:
I can remove it; I'll just also need to modify the startupTemplate script so that it moves the benchmark to a different file before installing the profile. That probably makes more sense.

Reply:
Done.